src/javascripts/modules/pills.js

```diff
@@ -9,31 +9,18 @@ export default class Pills {
   }
 
   handleLoad() {
-    clearTimeout(this.timeout)
     this.timeout = setTimeout(() => this.animatePills(), 500)
     this.releaseSqueezeTimeout = setTimeout(() => this.releasePillSqueeze(), 4500)
   }
 
   animatePills() {
-    if (this.el.classList.contains('yellow-pill')) {
-      this.el.classList.add('yellow-pill-roll')
-    } else if (this.el.classList.contains('more-pills')) {
-      this.el.classList.add('more-pills-roll')
-    } else if (this.el.classList.contains('purple-pills')) {
-      this.el.classList.add('purple-pills-roll')
-    } else if(this.el.classList.contains('title-tablet')) {
-      this.el.classList.add('title-tablet-roll')
-    } else if(this.el.classList.contains('title-pill-red')) {
-      this.el.classList.add('title-pill-red-bounce')
-    } else {
-      this.el.classList.add('pills-roll')
-    }
+    this.el.classList.add('pills-animating')
   }
 
   releasePillSqueeze() {
-    if (this.el.classList.contains('title-pill-red-bounce')) {
-      console.log('did i wait?')
-      this.el.classList.remove('title-pill-red-bounce')
+    var tempPill = this.el.children[1].children[0].children[0].children[0]
```

Comment:
This is heavily dependent on html source order. Are you just trying to get the red pill? If so you can use `querySelector` instead:
```
var tempPill = this.el.querySelector('.title-pill-red-bounce')
```
You can then remove the if block below. Or you could just do:
```
this.el.querySelector('.title-pill-red-bounce').classList.remove('title-pill-red-bounce')
```